### ISSUE TITLE
Create a demo config that uses all the v2 features

### ISSUE THREAD

**Issue by doug-walker** [AcademySoftwareFoundation/OpenColorIO#1195]
For testing purposes, it would be helpful to have a config that uses the new v2 features.  The ACES config is underway but it would be helpful to have something ASAP, even if it is just a toy/demo config.

This could be added to the repo under the tests/data directory.  

It could also be used for documentation purposes.

**Comment by KelSolaar** [comment]
That would be super useful, not only that but I would be keen to have some snippets, especially or the new display colorspace system that I could roll into the [ACES config generator](https://github.com/AcademySoftwareFoundation/OpenColorIO-Config-ACES/blob/e71d924701f23d78a77732ebb73dd233c8960bee/opencolorio_config_aces/config/generation/common.py#L510).

**Comment by zachlewis** [comment]
I have a notebook and plenty of snippets I've been cleaning up to share that produces this:
https://gist.github.com/zachlewis/3275d4893414d2c278487612121ecc68

(WARNING... I've never actually visually tested the above; IIRC, I'm computing my P3-D65 and DCI-P3 matrices from XYZ-E instead of XYZ-D65, which I'll have to fix).

It's a proof-of-concept that, among other things, maps some of the ACES-to-Rec.709 LUTs from the [ACES Gamut Mapping VWG resources repo](https://github.com/colour-science/aces-vwg-gamut-mapping-2020/tree/master/research/resources/ocio/configurations/aces_1.2/aces_vwg_gm) to various SDR displays, vis-a-vis View Transforms. I'm also using hypothetical show LUT + shot / neutral CTFs in looks applied to shared views. 


It's a real kitchen-sink affair -- I was trying to test the Python API's implementation of as many features as I could, simultaneously. 

I'll post a few useful snippets in a moment, and I'll get the notebook out shortly. It uses the `colour` library for matrix computations.

**Comment by zachlewis** [comment]
I've added some extremely-unsexy-but-hopefully-straightforward example [Python code](https://gist.github.com/zachlewis/3275d4893414d2c278487612121ecc68#file-build_test_ocio2_config-py) to the above gist to produce the config. Forgive the formatting, I copy-pasted from a notebook. Still haven't actually visually checked the config, but... what could possibly 🙈 go wrong? 

Worth noting:
 - This is an extremely simple use of Shared Views and Viewing Rules that treats every "sdr-video"-encoded display color space as approximately equal. For instance, the `K1S1` view for the `P3-D65`, `DCI-P3`, and `sRGB` display-view transforms are all adapted from the same Rec.709 LUT. Some might say adapting the same five LUTs to five separate displays is a little egregious. Others might call it economic. 
 - I can amend or share separately a more complex example that I also imagine 🙉 works. The more complex example articulates separate Rec.709, DCI-P3, and P3-D65 `ViewTransforms` for distinct variants of the K1S1, and uses a Viewing Rule to constrain the shared Rec.709 K1S1 view to just the "sRGB" and "Rec.709" display color spaces; the DCI-P3 and P3-D65 K1S1 Display / Views are implemented individually.
 - This is missing examples of the built-in ACES output transform stuff. I don't really know how to use the grading splines stuff.




I've also seeded a collection jar of _far_ sexier methods and snippets for anyone's use:
https://gist.github.com/zachlewis/66a54766208556a85a23e4701bd3aeb
